assets/elasticsearch/search.py

```
@@ -0,0 +1,47 @@
+import logging
+import sys
+import json
+
+from elasticsearch import Elasticsearch
+
+collection_name = "elasticsearch_clp_bench"
+
+es = Elasticsearch(
+    "http://localhost:9202", request_timeout=30, max_retries=10, retry_on_timeout=True
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)